### ISSUE TITLE
On angular 13 the generate command doesn't allow generating new cart components

### ISSUE THREAD

**Issue by emaas** [valor-software/ng2-charts#1373]
On Windows, with angular 13 and node 16 (clean new project).
Running `ng generate ng2-charts:radar charts/radar-chart` does not work.
It shows:
`An unhandled exception occurred: Schematic "radar" cannot resolve the factory.`
With the angular-errors.log:
```[error] Error: Schematic "radar" cannot resolve the factory.
    at SchematicEngineHost.createSchematicDescription (C:\project\node_modules\@angular-devkit\schematics\tools\file-system-engine-host-base.js:183:19)
    at SchematicEngine.createSchematic (C:\project\node_modules\@angular-devkit\schematics\src\engine\engine.js:228:38)
    at CollectionImpl.createSchematic (C:\project\node_modules\@angular-devkit\schematics\src\engine\engine.js:80:29)
    at GenerateCommand.getSchematic (C:\project\node_modules\@angular\cli\models\schematic-command.js:150:27)
    at GenerateCommand.initialize (C:\project\node_modules\@angular\cli\models\schematic-command.js:63:36)
    at async GenerateCommand.initialize (C:\project\node_modules\@angular\cli\commands\generate-impl.js:20:9)
    at async GenerateCommand.validateAndRun (C:\project\node_modules\@angular\cli\models\command.js:121:22)
    at async Object.runCommand (C:\project\node_modules\@angular\cli\models\command-runner.js:224:24)
    at async default_1 (C:\project\node_modules\@angular\cli\lib\cli\index.js:70:31)
```

The same problem persists when downgrading angular cli to 12.2.15

My current dependencies are:
```"dependencies": {
    "@angular/animations": "~13.1.0",
    "@angular/cdk": "^13.1.1",
    "@angular/common": "~13.1.0",
    "@angular/compiler": "~13.1.0",
    "@angular/core": "~13.1.0",
    "@angular/forms": "~13.1.0",
    "@angular/material": "^13.1.1",
    "@angular/platform-browser": "~13.1.0",
    "@angular/platform-browser-dynamic": "~13.1.0",
    "@angular/router": "~13.1.0",
    "chart.js": "^3.7.0",
    "ng2-charts": "^3.0.6",
    "rxjs": "~7.4.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~13.1.2",
    "@angular/cli": "~12.2.0",
    "@angular/compiler-cli": "~13.1.0",
    "@types/jasmine": "~3.10.0",
    "@types/node": "^12.11.1",
    "jasmine-core": "~3.10.0",
    "karma": "~6.3.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.1.0",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "~1.7.0",
    "typescript": "~4.5.2"
  }
```

When using line instead or radar, the error changes to:
`ENOENT: no such file or directory, open 'C:\project\node_modules\ng2-charts\schematics\ng-generate\line\files\__path__\__name@dasherize@if-flat__\__name@dasherize__.component.html.template'`

**Comment by emaas** [comment]
Found out that the template file are actually not available in the ng-generate folders in the dist directory.
Solution seems to be just to copy over those files after we are done building.
This seems to work, but if anyone knows a more elegant solution, please go ahead and use that.

**Comment by santam85** [comment]
Should be fixed in v3.0.8
